src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -175,6 +176,23 @@ public static Metric NotSupported()
             return _notSupported;
         }
 
+        /// <summary>
+        /// A metric for when a FHIR resource has been saved.
+        /// </summary>
+        /// <param name="resourceType">The type of FHIR resource that was saved.</param>
+        /// <param name="resourceOperation">The operation performed on the FHIR resource.</param>
+        public static Metric FhirResourceSaved(ResourceType resourceType, ResourceOperation resourceOperation)
+        {
+            return new Metric(
+                $"FhirResourceSaved",
+                new Dictionary<string, object>
+                {
+                    { _nameDimension, $"{resourceType}FhirResource{resourceOperation}" },
```

Comment:
Will this output the ordinal of the enum, of the Name of the enum?

Reply:
Should be the name.  This is an implicit ToString()